designsafe/apps/applications/views.py

```diff
@@ -398,8 +398,8 @@ def call_api(request, service):
                 return HttpResponse('Unexpected service: %s' % service, status=400)
         except AgaveException as e:
             logger.error('Failed to execute {0} API call due to AgaveException={1}'.format(
-                service, e.message))
-            return HttpResponse(json.dumps(e.message), content_type='application/json',
+                service, e))
+            return HttpResponse(json.dumps(e), content_type='application/json',
```

Comment:
Just testing a bit and see that `'AgaveException' is not JSON serializable`.  Looks like it should be `HttpResponse(e, ...)` for here and on Line 419.